### PR TITLE
[DO NOT MERGE YET] Openstack controller integration docs

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
**Ticket** : https://issues.newrelic.com/browse/NR-154951
**Description**: Openstack Controller - created integration documentation for openstack controller.
**Preview image**:

screencapture-localhost-8000-docs-infrastructure-host-integrations-host-integrations-list-openstack-controller-integration-2023-08-30-20_11_08

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,722 @@
...
+
+You can install the infrastructure agent two different ways:
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview).
+* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/).
+
+## Installation
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx:31]:
```suggestion
## Configure nri-openstack
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,722 @@
...
+```
+
+3. Rename the file `config/os-config.json.template` to `config/os-config.json` and edit the file as mentioned below
+
+    * `nr_agent_home`. you need to enter the parent directory of the **config/** folder from  your cloned repository.
+    * `keystone_url`. Run the below command and display the all Endpoints with name. Copy and use the  keystone endpoint for keystone_url.
+      ```shell
+      openstack catalog list
+      ```
+    * `keystone_api_version` and `neutron_api_version`. Use the below command to find the api version.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx:49]: Could you please add example of <endpoint URL>

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,722 @@
...
+              osServiceType: resource_providers
+            apis:
+              - name: OSResourceProvider
+                commands:
+                  - run: "mkdir -p ${var:nrAgentHome}/logs 2>/dev/null ; ${var:nrAgentHome}/scripts/flex-osmetrics.sh ${var:osServiceType} 2>>${var:nrAgentHome}/logs/os-mon-${var:osServiceType}-err.log"    
+      ```
+      </Collapser>
+</CollapserGroup>
+
+### Restart the New Relic infrastructure agent
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx:688]:
```suggestion
## Restart the New Relic infrastructure agent
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,722 @@
...
+
+Before you can start reading your data, use the instructions in our [infrastructure agent docs](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) to restart your infrastructure agent.
+
+```shell
+sudo systemctl restart newrelic-infra.service
+```
+###  Monitor your application
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx:695]:
```suggestion
## Monitor your application
```

**Review by pkosanam — CHANGES_REQUESTED** [review]

**Review by akristen — APPROVED** [review]
This looks good! Going ahead to merge. :)

**Comment by akristen** [comment]
I've approved this PR, but reached out to @RamanaReddy8801 and confirmed that it's not ready to merge. Reach out to us in #help-documentation and we'll get it merged when ready :)

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,731 @@
...
+
+    * To find the `keystone_api_version`, run the below command.
+      ```shell
+      curl -i http://<HOST_IP>/identify
+      ```
+    * To find the `glance_api_version`, run the below command.
+      ```shell
+      curl -i http://<HOST_IP>/image
+      ```
+    * Update `admin user password`, you need to enter the openstack login password.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration.mdx:60]: is it `admin user password` or `ADMIN_PASSWORD` ?

**Review by pkosanam — CHANGES_REQUESTED** [review]

**Review by pkosanam — APPROVED** [review]
Looks good to me.. 👍

**Comment by akristen** [comment]
Eng triggered a manual build, which was successful. Merging

**Pull request merged**